packages/fluent/scss/menu/_layout.scss

```diff
@@ -178,7 +189,8 @@
         }
 
         .k-separator {
-            margin: var( --kendo-menu-separator-margin, $kendo-menu-separator-margin ) 0;
+            margin-block: $kendo-menu-separator-margin-inline;
+            margin-inline: 0;
             height: 0;
             border-width: 0;
             border-top-width: var( --kendo-menu-separator-width, $kendo-menu-separator-width);
```

Comment:
The syntax here should be: border-top-width: var( --kendo-menu-separator-width, #{$kendo-menu-separator-width});